### PR TITLE
feat(CommunityOverview): Add Community Overview banners

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #5978

### What does the PR do

Add new banners for CommunityOverviewSettings

### Screenshot of functionality
Снимок экрана 2022-06-09 в 12 09 48

### CODE REVIEW

File: ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml [hunk 1]
```diff
@@ -8,10 +8,19 @@ import StatusQ.Controls 0.1
...
+
+import utils 1.0
 
 StackLayout {
     id: root
 
+    property bool hasAddedContacts: false
+    property var activeCommunity
+    property var communitySectionModule
+    property var store
```
Comment (osmaczko) [inline ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml:21]: According to QML core architecture panels should not have any dependency to a store or a view.
https://doc-snapshots.qt.io/neptune3ui/neptune3ui-application-architecture.html
https://hackmd.io/@status-desktop/main/https%3A%2F%2Fhackmd.io%2F%40status-desktop%2FB1eOaf-nd

The solution would be to propagate all banner signals to `CommunitySettingsView`. 
The flow should look as follows `CommunityOverviewSettingsPanel`-signals->`CommunitySettingsView`-calls->`store`-triggers->`popups`

File: ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml [hunk 2]
```diff
@@ -8,10 +8,19 @@ import StatusQ.Controls 0.1
...
+import "../../popups/community"
+
+import utils 1.0
 
 StackLayout {
     id: root
 
+    property bool hasAddedContacts: false
+    property var activeCommunity
+    property var communitySectionModule
```
Comment (osmaczko) [inline ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml:20]: The backend object should not be used in the Panel (see another comment mentioning QML core architecture) and ideally, backend objects should be accessed only through stores.

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml [hunk 3]
```diff
@@ -34,6 +34,7 @@ StatusAppTwoPanelLayout {
     property var rootStore
     property var community
     property var chatCommunitySectionModule
+    property bool hasAddedContacts: false
```
Comment (osmaczko) [inline ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:37]: This property is not set from anywhere (always false) :thinking: Was it intended?

**Review by osmaczko — CHANGES_REQUESTED** [review]

**Comment by elina2015** [comment]
@borismelnik I found some small issues. Could you kindly check them?
1. The spaces between blocks differ from design:
image
Design:
image
2. The text for the backup community key differs from design:
image
Design:
image

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml (hunk 3 above)
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:37]: @osmaczko no I think I set it :)

File: ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml (hunk 2 above)
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml:20]: @osmaczko  I forgot, will fix

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml (hunk 3 above)
Comment (osmaczko) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:37]: hmm.. can you point me to the line?

Comment (borismelnik) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:37]: Yeah i lost it for commit :D

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml [hunk 4]
```diff
@@ -146,6 +149,22 @@ StatusAppTwoPanelLayout {
                         errorDialog.open()
                     }
                 }
+
+                onInviteNewPeopleClicked: {
+                    Global.openPopup(inviteFriendsToCommunityPopup, {
```
Comment (osmaczko) [inline ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:154]: Where is `inviteFriendsToCommunityPopup` defined? Dynamic scoping is [anipattern](https://github.com/status-im/StatusQ/blob/master/src/contributing.md#scope) we are trying to get rid off, could you please fix that?

**Review by osmaczko — COMMENTED** [review]
Looks much better now, thanks for the fixes.

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml (hunk 4 above)
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:154]: @osmaczko  That used everywhere, are you sure that I should did it here?

Comment (borismelnik) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:154]: Take your popup and give me your approve :D

Comment (borismelnik) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:154]: but I think its a good idea with `issue`

Comment (borismelnik) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:154]: https://github.com/status-im/status-desktop/issues/6060

**Review by osmaczko — APPROVED** [review]

**Review by elina2015 — APPROVED** [review]
Tested and Approved

**Pull request merged**